### DISCUSSION TITLE
How to map a jsonSchema containing allOf element with $ref to another subSchema, towards a target

### DISCUSSION THREAD

**Discussion by savbran** [atlasmap/atlasmap#3722] [General]
Hi,

As reported in subject, I have a jsonSchema (let's say jsA) containing a  "properties" element with several properties (strings, objects, etc). At the same level of properties then I have an "allOf" element with $ref to another json subSchema (let's say jsB).

When I import the jsA in AtlasMap UI, I can't see the elemens contained in jsB. Is there a way to map JsA and also jsB towards a target?

Thanks in advance.

**Comment by igarashitm** [comment]
`allOf` in JSON schema is not yet supported - https://github.com/atlasmap/atlasmap/issues/24
And multiple JSON schema referencing one from another is not supported yet either. You can file an enhancement request. And contributions are always welcome!
